.github/scripts/get_workflow_type.py

```diff
@@ -49,47 +48,35 @@ def is_exception_branch(branch: str) -> bool:
 
 
 def get_workflow_type(issue: Issue, username: str) -> str:
-    user_list = issue.get_comments()[0].body.split("\r\n")
     try:
-        run_option = issue.get_comments()[1].body.split("\r\n")[0]
-    except Exception as e:
-        run_option = "single"
-
-    if user_list[0] == "!":
-        # Use old runners for everyone
-        return WORKFLOW_TYPE_LABEL
-    elif user_list[1] == "*":
-        if run_option == WORKFLOW_TYPE_BOTH:
-            # Use ARC runners and old runners for everyone
-            return WORKFLOW_TYPE_BOTH
-        else:
-            # Use only ARC runners for everyone
-            return WORKFLOW_TYPE_RG
-    elif username in user_list:
-        if run_option == WORKFLOW_TYPE_BOTH:
-            # Use ARC runners and old runners for a specific user
-            return WORKFLOW_TYPE_BOTH
+        user_list = issue.get_comments()[0].body.split("\r\n")
```

Comment:
I'm surprised to see `\r` included in the splitter.

Seems like a subtle bug ready to fire if the wrong tool/OS updates the comment

What do you think about spliting it on just `\n`, and then trimming each resulting entry for `\r`?

Reply:
I double-checked, we don't even need "\r\n".